spark/dl/src/main/scala/com/intel/analytics/bigdl/optim/Metrics.scala

```diff
@@ -21,6 +21,9 @@ import org.apache.spark.{Accumulable, Accumulator, SparkContext}
 
 import scala.collection.mutable.{ArrayBuffer, Map}
 
+/**
+ * Set metrics of local models on each executor
```

Comment:
Performance metrics for the training process. Beyond collect local metrics(e.g. throughput) in driver node, it can also be used to collect distributed metrics(e.g. time of some steps among the workers). The is useful for performance analysis.